client/components/dashboard-activities.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { StyleSheet, Text, View, Image, ScrollView} from 'react-native'
+import {connect} from 'react-redux'
+import { Container, Tabs,Tab, ScrollableTab } from 'native-base';
+
+const UserActivities = (props) => {
+    const {activities} = props
```

Comment:
I believe the getActivities thunk needs to be dispatched here. Otherwise there will never be activities.

Reply:
Agreed. I am merging this pull request anyway to prevent conflicts with an upcoming pull requests from me and Lamine.